src/backend/tests/it/get_users.rs

```diff
@@ -0,0 +1,186 @@
+use std::time::{Duration, UNIX_EPOCH};
+
+use crate::utils::{
+    mock::{CALLER, ISSUER_CANISTER_ID, VC_HOLDER, VP_JWT},
+    pocketic::{query_call, setup, update_call},
+};
+use candid::Principal;
+use ic_verifiable_credentials::issuer_api::CredentialSpec;
+use pocket_ic::PocketIc;
+use shared::types::user_profile::{
+    AddUserCredentialError, AddUserCredentialRequest, GetUsersRequest, GetUsersResponse, OisyUser,
+    UserProfile,
+};
+
+fn create_users(pic_setup: &(PocketIc, Principal), start: u8, end: u8) {
+    for i in start..=end {
```

Comment:
advance maybe the timers sometimes in the test to not have all the users created with the same timestamp?

Reply:
I do that outside this function when needed.

For example, in `test_list_users_returns_filtered_users_by_updated` I create two sets of 10 users with different timestamps.

This way the test controls the time and can manage it.

* Do you see a reason why making sure each user has a different timestamp is important?
* Would you then assume that some X time has passed after calling `create_users`.

Reply:
> Do you see a reason why making sure each user has a different timestamp is important?

I had once issue because pocket-ic was too fast and therefore had to explicitely add some delay to test both "fast" and "slow" scenarios. I was thinking that here it could be particular useful because the maps are ordered by timestamps.

> I do that outside this function when needed.

If we cover use cases where all users do not have the same time stamp it's good, just felt like it would be nice to have this tested explicitely for above reasons.